core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -69,7 +69,7 @@ protected function addCssAttributesToTemplate(Template $template, string $templa
         $template->class = trim($templateName.' '.($data[1] ?? ''));
         $template->cssID = !empty($data[0]) ? ' id="'.$data[0].'"' : '';
 
-        if (\is_array($classes)) {
+        if (!empty($classes)) {
```

Comment:
```suggestion
        if (!empty($classes) && \is_array($classes)) {
```

Reply:
Do we have to keep it? `$classes` can only be an `array` or `null`, since it's a typed argument. The `\is_array` check should be superfluous, since an exception would occurs, if you did not pass an array (or null) as the last argument to this function.

Reply:
You are right.